requirements_qa.txt

```diff
@@ -1 +1,3 @@
-pylint==2.6.0
\ No newline at end of file
+pylint==2.6.0
+numpy==1.20.1
```

Comment:
no, that should not be here. students will have to add these dependencies on their own

Reply:
fixed